example/setup.py

```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env python
 
 from setuptools import setup
+from setuptools.extension import Extension
 from distutils.command.build import build as _build
 
+# Option "-D_hypot=hypot" is mandatory for mingw64
+extensions = [Extension('cysignals_example',
+                        ['cysignals_example.pyx'],
+                        extra_compile_args=['-D_hypot=hypot'])]
```

Comment:
What's up with `hypot`? The example code doesn't involve `hypot` at all, so why is this needed?

Reply:
short story :
````
{{{In file included from c:/mingw-w64/x86_64-7.2.0-posix-seh-rt_v5-rev1/mingw64/lib/gcc/x86_64-w64-mingw32/7.2.0/include/c++/math.h:36:0,
                 from c:\Python27-x64\include/pyport.h:325,
                 from c:\Python27-x64\include/Python.h:58,
                 from cysignals_example.cpp:24:
c:/mingw-w64/x86_64-7.2.0-posix-seh-rt_v5-rev1/mingw64/lib/gcc/x86_64-w64-mingw32/7.2.0/include/c++/cmath:1136:11: error: '::hypot' has not been declared
   using ::hypot;
```

Long story :
https://ci.appveyor.com/project/vinklein/cysignals/build/1.0.194

Reply:
That is likely a bug in mingw64.

Can you send me the pre-processed source file (created with `-save-temps`)?

Reply:
And maybe also the file `c:/mingw-w64/x86_64-7.2.0-posix-seh-rt_v5-rev1/mingw64/lib/gcc/x86_64-w64-mingw32/7.2.0/include/c++/cmath`

Reply:
It's actually a Python bug: https://github.com/python/cpython/pull/880

Reply:
In master, I pushed a commit to stop using `<math.h>` in the example. Hopefully the issue will be gone then.

Reply:
Ok nice

Reply:
I don't know why but it look like `<math.h>` is still, called https://ci.appveyor.com/project/vinklein/cysignals/build/job/no6rwrud4sojkauv.